pkg/apis/camel/v1alpha1/types.go

```diff
@@ -125,6 +125,7 @@ type IntegrationContext struct {
 // IntegrationContextSpec --
 type IntegrationContextSpec struct {
 	Dependencies  []string            `json:"dependencies,omitempty"`
+	Classpath     []ClasspathEntry    `json:"classpath,omitempty"`
```

Comment:
Should we move this to "status"? I don't think it's something the user should set. It's computed from the dependencies.